ckanext/depositar_theme/templates/home/index/idx_stats.html

```diff
@@ -0,0 +1,40 @@
+{% set formatCntList = h.get_format_count() %}
+
+<div class="idx stats">
+  <div class="container">
+    <div class="stat-data">
+      <div class="imp-data">
+        <div class="imp-item">
+          <h1>{{ _('Datasets') }}</h1>
+          <a href="{{ h.url_for('/dataset') }}" class="btn">
+            <p class="cnt">{{ h.get_site_statistics()['dataset_count'] }}</p>
+          </a>
+        </div>
+        <div class="imp-item">
+          <h1>{{ _('Projects') }}</h1>
+          <a href="{{ h.url_for('/organization') }}" class="btn">
+            <p class="cnt">{{ h.get_site_statistics()['organization_count'] }}</p>
+          </a>
+        </div>
+        <div class="imp-item">
+          <h1>{{ _('Downloads') }}</h1>
+          <p class="cnt">{{ h.get_download_count() }}</p> 
+        </div>
+      </div>
+      <div class="sec-data">
+        {% for formatCnt in formatCntList[:7] %}
+        <div class="sec-item">
+          <h5>{{ formatCnt[0] }}</h5>
+          <a href="{{ h.url_for('/dataset?res_format=' + formatCnt[0]) }}" class="btn">
```

Comment:
改為 Flask endpoint (route) name

```suggestion
          <a href="{{ h.url_for('dataset.search', res_format=formatCnt[0]) }}" class="btn">
```